P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1326 @@
+      </list></p>
+    <p>CMC supports multimodal expression combining text, images, sound. Whereas early CMC systems
+      (e.g. Internet Relay Chat, <soCalled>IRC</soCalled> for short, the Usenet
+      <soCalled>newsgroups</soCalled>, or even the Unix <name>talk</name> system)
+      were completely ASCII-based, most CMC applications now permit combining media formats (e.g. written or spoken
+      language with graphic icons and images) and mixing communication technologies on one platform (e.g. combined use of an audio connection, a chat system, and a
+      3D interface in which users control a virtual avatar as in many multiplayer online computer
+      games or in virtual worlds).</p>
+  </div>
+  <div type="div2" xml:id="CMCUnits">
+    <head>Basic Units of CMC</head>
+    <p>This section describes the encoding mechanisms for the basic units of CMC and for their
+      combined use to encode CMC data.</p>
+    <p>We use the term <term>basic CMC unit</term> to refer to a communication produced by an interlocutor to initiate or contribute to an ongoing
+      CMC interaction or joint CMC activity. Contributions
+      to an ongoing interaction are produced to perform a move to develop
+      the interactional sequence, for instance to respond in chats or forum discussions. Contributions to joint
+      CMC activities may not all be directly interactional; some may be part of a collaborative
+      project of the involved individuals. Such collaboration could involve editing activities in a shared text editor
+      or whiteboard in parallel with an ongoing CMC interaction (chat, audio conversation, or
+      audiovideo conference) in the same CMC environment in which these editing activities are
+      discussed by the participants.</p>
+    <p>Basic units of CMC can be described according to three criteria: 
+      <list rend="numbered">
+        <item>the temporal properties
+          of when these contributions are produced by their creators, transmitted via CMC systems, and
+          made accessible for the recipients;</item>
+        <item>the modality of the unit as a whole, whether verbal or noverbal;</item>
+        <item>for verbal units: whether the unit is expressed in the written or
+          spoken mode.</item>
+      </list>
+    A taxonomy of basic CMC units resulting from these criteria is given in the following figure.
+    </p>
+    <figure xml:id="cmcunits-taxonomy">
+      <graphic url="Images/cmcunits-taxonomy.png"/>
+      <head>Taxonomy of basic CMC units according to <ptr type="cit" target="#BIB_CMC_Core"/></head>
+    </figure>
+    <p>The most important distinction in <ref
+    target="#cmcunits-taxonomy">the CMC taxonomy</ref>
+      concerns the temporal nature of units exchanged via CMC technologies. The left part of the
+      taxonomy describes units that are performed (by a producer) and perceived (by a recipient) as
+      a continuous stream of behaviour. Units of this type can be performed as</p>
```

Comment:
bahavior (for consistency reasons)

Reply:
I think in reality the problem are the instances with the AE spelling, right?